### DISCUSSION TITLE
New to PlexKodiConnect and so I have a few questions.

### DISCUSSION THREAD

**Discussion by OzzieAusbourne** [croneter/PlexKodiConnect#2003] [Q&A] [answered]
Firstly @croneter thankyou so much for your amazing work on PlexKodiConnect. I was looking at having to migrate to Jellyfin to avoid plexplayer app even thought I did not really want to give up on plex server so this is perfect for me.

I got PlexKodiConnect all configured and installed on my test device and now I can do the full configuration for other devices but I have a few questions.

1) I only run my plex server on my LAN and do not have any remote management configured and I also do not login to my server with a plex account (so the console always says my server is unclaimed) by using PlexKodiConnect am I potentially exposing myself to security risks as a result of this ? I manually entered my Plex server IP for PlexKodiConnect setup.

2) In Plex Server I do not have my LAN range specificed in the List of IP addresses and networks that are allowed without auth, is this potentially a security issue when using PlexKodiConnect (or even generally).

3) During setup of PlexKodiConnect it warned me that I should be using https not http however only HTTP would work.

4) I just wanted to check that PlexKodiConnect will detect new media when it is added, I left the synch options as they are so enable constant background synch is on which I assume works like plex option to rescan when it detects changes ?

If there is anything else I should lock down in Plex or PlexKodiConnect to keep it to LAN only then please let me know.

**Comment by iirishguyi** [comment]
Wow......are you sure you didn't miss anything?? 😆 Or is this just in order to keep your business a float?...... so your customers are satisfied

**Comment by OzzieAusbourne** [comment]
> Wow......are you sure you didn't miss anything?? 😆 Or is this just in order to keep your business a float?...... so your customers are satisfied

Huh ?

**Comment by croneter** [comment, marked as answer]
Glad you like PKC, @OzzieAusbourne!

1. Nope, PKC is not an "additional" security risk
2. This is a very complex question - there's a lot you could potentially configure wrong in your network setup, unfortunately. However, this is not PKC's "issue". 
3. That's because you're only using your PMS within your LAN with no remote stuff configured - hence the TLS certificates for HTTPS do not work from the PMS side. This means that an attacker that has already gained access to your LAN / home network could listen in on the traffic between PKC and the PMS. 
4. Yes, PKC detects new media automatically. 

Nothing to "lock down" on the PKC side I could think of
